### PR TITLE
Add build on F33 and Centos 8

### PR DESCRIPTION
**Pull request by mscherer**

### CODE REVIEW

**Comment by mscherer** [comment]
So, I am not sure how ready we are for the el8 rpms, nor the F32 ones. I am trying to get ride of Fedora 30 builder slowly.

**Comment by deepshikhaaa** [comment]
recheck

**Comment by mscherer** [comment]
This PR was split in 2, with #53 
I will remove the commit once it is merged.

File: build-gluster-org/jobs/rpm.yml [hunk 1]
```diff
@@ -7,9 +7,15 @@
       - fedora:
           build_flag: fedora-30-x86_64
```
Comment (deepshikhaaa) [inline build-gluster-org/jobs/rpm.yml:8]: Let's remove F30 configuration. Also, didn't we upgrade to fedora32?

Comment (mscherer) [reply build-gluster-org/jobs/rpm.yml:8]: F32 didn't seems to support F30 anymore, but yeah, let's remove that.

**Comment by mscherer** [comment]
Closing since I opened another PR with newer OS

**Pull request closed without merging**